src/Compute/Compute/help/Add-AzVMDataDisk.md

```diff
@@ -17,16 +17,17 @@ Adds a data disk to a virtual machine.
 ```
 Add-AzVMDataDisk [-VM] <PSVirtualMachine> [[-Name] <String>] [[-VhdUri] <String>] [[-Caching] <CachingTypes>]
  [[-DiskSizeInGB] <Int32>] [-Lun] <Int32> [-CreateOption] <String> [[-SourceImageUri] <String>]
- [-DiskEncryptionSetId <String>] [-DeleteOption <String>] [-DefaultProfile <IAzureContextContainer>]
- [<CommonParameters>]
+ [-DiskEncryptionSetId <String>] [-DeleteOption <String>] [-SourceResourceId <String>]
+ [-DefaultProfile <IAzureContextContainer>] [-ProgressAction <ActionPreference>] [<CommonParameters>]
```

Comment:
ProgressAction is added unnecessarily again , this happens often these days.

Reply:
@NoriZC  generating md file adds "-ProgressAction". 
Should I be removing them?

Reply:
This is a known issue caused by PlatyPS in PowerShell 7.4. You can either remove it in pr or we will remove it in our release process.